part-1/part-1.md

```diff
@@ -163,51 +181,54 @@ For example:
 
-We will review some of these combinations and their use-cases in a later article. And dig deeper in advanced RxJS course.
+We will review some of these combinations and their use-cases in a later chapter. And dig deeper in advanced RxJS course.
 
-## Now lets review the main differences between Promises and Observables you should be aware of (or they will bite you some time)erence
+## Now lets review the main differences between Promises and Observables you should be aware of (or they will bite you some time later)
 
-## Promises **vs** Observables
+### Promises **vs** Observables
 
 At first glance — Observables are just advanced Promises: Promises emits one value and complete (resolve), Observables emit 0, one or many values and complete as well (emit and complete are different actions). For HTTP service in AngularJS (where it used Promises) and Angular (where it uses Observables) provides only one value — so seems both frameworks work very similar in this case.
 
 ![](RackMultipart20210318-4-1e6dc56_html_fc59ddf18c80e00b.png)
 
-but if you application is doing something more then 'Hello world' — plz pay attention to the differences
+but if you application is doing something more then 'Hello world' — please pay attention to the differences.
 
-## **#1 Eager vs Lazy**
+### #1 Eager vs Lazy
 
-Take a look at example below: ![](RackMultipart20210318-4-1e6dc56_html_533e70b36f8382d3.png)
+Take a look at the example below: ![](RackMultipart20210318-4-1e6dc56_html_533e70b36f8382d3.png)
 
-When I call saveChanges method — the first example with Promise-wrapped request will work as expected. But in seconds Observable-wrapped example nothing will happen because Observables are lazy-evaluated while Promises are eager-evaluated.
+When I call the `saveChanges` method — the first example with `Promise`-wrapped request will work as expected. But in seconds `Observable`-wrapped example nothing will happen because `Observables` are lazily-evaluated while `Promises` are eagerly-evaluated.
 
-This means that Promises doesn't care whether they have some subscribers to get their result or not. But Observables (to be precise — cold Observable) will be cold only if we subscribe to them. In the case above you should subscribe to Observable returned by **saveChanges** function.
+This means that Promises doesn't care whether they have any consumers that wait for their result or not, they will emit the value regardless. But `Observables` (to be precise — *cold* `Observables`) will emit only if we subscribe to them. In the case above you should subscribe to the `Observable` returned by **saveChanges** function.
```

Comment:
Is it worth adding a brief high level overview of what hot observables are (since cold observables are mentioned here)? It might be out of scope here, maybe as part of different chapter?

cc: @LayZeeDK

Reply:
@williamjuan027 I've opened a discussion in our Discord chat.

Reply:
I suggest we have a different chapter on that